hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerStateManagerImpl.java

```diff
@@ -334,8 +347,12 @@ ContainerInfo getMatchingContainer(final long size, String owner,
     throw new UnsupportedOperationException("Not yet implemented!");
   }
 
-  public void removeContainer(final HddsProtos.ContainerID id) {
-    containers.removeContainer(ContainerID.getFromProtobuf(id));
+  @Override
+  public void removeContainer(final HddsProtos.ContainerID id)
```

Comment:
So there is no easy way to test such changes?

Reply:
we can verify the state of memory and RocksDB in Unit Test.